.travis.yml

```diff
@@ -58,13 +58,24 @@ script:
 
 env:
   global:
-  # OSC_CREDENTIALS are kept secret
-  # In order to update them, run:
-  #   travis encrypt OSC_CREDENTIALS=user:password
-  # more info at: http://docs.travis-ci.com/user/environment-variables/#Secure-Variables
-  - OBS_REPO=Virtualization:containers:Portus
-  - OBS_BRANCH=master
-  - secure: "SgH8Wvm6n71GTZqTk2Ww/vLg3KK1QhgBMRSs+85TFemJvUDEWLKtANGI2szlqJOfOEWqHZ4rdzgMamtYfGmOZvjUBebaIXlYMcPyKSmcQZX/80R1/v7KFyhP2vqrb+ok/rK6lLXX06jDhohUS04vnx4i1EtoxH62dKImdlZsjzI="
+    # OSC_CREDENTIALS are kept secret
+    # In order to update them, run:
+    #   travis encrypt OSC_CREDENTIALS=user:password
+    # more info at: http://docs.travis-ci.com/user/environment-variables/#Secure-Variables
+    - OBS_REPO=Virtualization:containers:Portus
+    - OBS_BRANCH=master
+    - secure: "SgH8Wvm6n71GTZqTk2Ww/vLg3KK1QhgBMRSs+85TFemJvUDEWLKtANGI2szlqJOfOEWqHZ4rdzgMamtYfGmOZvjUBebaIXlYMcPyKSmcQZX/80R1/v7KFyhP2vqrb+ok/rK6lLXX06jDhohUS04vnx4i1EtoxH62dKImdlZsjzI="
+
+    # DB options for both adapters and Travis.
+    - PORTUS_DB_USERNAME=travis
+    - PORTUS_DB_POOL=10
+
+    # Enable debug logger for VCR cassettes.
+    - PORTUS_VCR_LOGGER=true
```

Comment:
Are we gonna keep this enabled?

Reply:
Will remove this. This was part of me going mad 😄

Reply:
👍